src/react-apps/applications/runtime/src/utils/networking.ts

```diff
@@ -47,6 +47,18 @@ export async function put(
   }
 }
 
+export async function restDelete(
```

Comment:
what is restDelete? Rename it to delete

Reply:
delete is a reserved word/name in javascript.